Frends.Facebook.Request/Frends.Facebook.Request/Definitions/Result.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace Frends.Facebook.Get.Definitions;
+﻿namespace Frends.Facebook.Request.Definitions;
 
 /// <summary>
```

Comment:
Now that we are building Http wrapper it might be better to return the StatusCode as well or instead of the Success.